app/controllers/api/harvests_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Api
+  # Initiate a harvest via the API (e.g. from Airflow)
+  class HarvestsController < ApiController
+    delegate :t, to: :I18n
+
+    include NdjsonUploadConcern
+
+    rescue_from ActionController::ParameterMissing, with: :no_url
```

Comment:
I'm not a huge fan of `rescue_from` because it makes it hard to follow the program flow.  How about wrapping the lines that raise these with `begin ... rescue` instead?

Reply:
This was my attempt to placate rubocop's branch condition rule, haha. I'm OK with using `begin`; will change it